README.md

```diff
@@ -35,7 +35,7 @@ If you want to build for yourself, just install [DMD](http://dlang.org/download.
 
 ### Arch Linux
 
-Михаил Страшун (Dicebot) maintains a dub package of the latest release in `Community`, for [x86_64](https://www.archlinux.org/packages/community/x86_64/dub/) and [i686](https://www.archlinux.org/packages/community/i686/dub/).
+Dub is in the [official repository](https://www.archlinux.org/packages/community/x86_64/dub/). AUR packages are also available: Михаил Страшун (Dicebot) maintains a dub package of the latest release in `Community`, for [x86_64](https://www.archlinux.org/packages/community/x86_64/dub/) and [i686](https://www.archlinux.org/packages/community/i686/dub/).
```

Comment:
The original text is correct, as it references the [community] package not an AUR package.

But two things are incorrect in the text.
Firstly, I'm the maintainer of all the D packages in the official repository (since 2017-09-10). I forgot to check all the different projects if they mentioned the maintainer, so this would probably be good to change, or just remove the mention.
Secondly, there is no longer an i686 package, as Arch Linux deprecated 32-bit back in 2017-11-08.

Reply:
ah yes you're correct - i was getting 'community' and 'aur' mixed up.